docs/sources/setup/install/helm/install-scalable/_index.md

```diff
@@ -13,70 +13,83 @@ keywords:
 <!-- vale Grafana.Quotes = NO -->
 <!-- vale Grafana.Quotes = YES -->
 
-This Helm Chart installation runs the Grafana Loki cluster within a Kubernetes cluster.
+This Helm Chart deploys Grafana Loki on Kubernetes.
 
-If object storge is configured, this chart configures Loki to run `read` and `write` targets in a [scalable mode]({{< relref "../../../../get-started/deployment-modes#simple-scalable" >}}), highly available architecture (3 replicas of each) designed to work with AWS S3 object storage. It will also configure meta-monitoring of metrics and logs.
+This chart configures Loki to run `read`, `write`, and `backend` targets in a [scalable mode]({{< relref "../../../../get-started/deployment-modes#simple-scalable" >}}), highly available architecture designed to work with AWS S3 object storage. The chart also supports self-monitoring or meta-monitoring by deploying Grafana Agent to monitor Loki itself, by scraping its metrics and logs. 
```

Comment:
Do we want to mention the other object stores as well?

Reply:
What were you thinking, something like mentioning that they could edit the chart to use a different store?  Because the example in this procedure is for AWS S3.

Reply:
Ah, I read that wrong. It also mentions the other stores later on so please ignore this comment.